Dockerfile

```diff
@@ -41,11 +41,13 @@ RUN ln -fs /usr/lib/x86_64-linux-gnu/libldap.so /usr/lib/ \
 	&& docker-php-ext-install ldap \
 	&& a2enmod rewrite ldap headers
 
-# Install nodejs
-RUN curl -sL https://deb.nodesource.com/setup_4.x | bash - \
-	&& apt-get install -y nodejs \
+# Install nodejs-legacy, npm and karma
+RUN apt-get update && apt-get install -y nodejs-legacy npm \
+	&& curl -sL https://deb.nodesource.com/setup_4.x | bash - \
+	&& apt-get upgrade -y nodejs \
 	&& npm update -g npm \
 	&& npm install -g bower gulp protractor jscs jshint typescript typings \
+	&& npm install gulp-util karma karma-jasmine karma-phantomjs-launcher --save-dev \
```

Comment:
Install all the different utilities needed for Karma Unit testing with PhantomJS